prompb/types.proto

```diff
@@ -56,6 +56,8 @@ message ReadHints {
   string func = 2;    // String representation of surrounding function or aggregation.
   int64 start_ms = 3; // Start time in milliseconds.
   int64 end_ms = 4;   // End time in milliseconds.
+  repeated string groups = 5; // List of label names of any fields used in a group-by aggregation
```

Comment:
Full stops. And there's no such thing as a "group-by".

Reply:
sure, will change it to `group_by`

Reply:
`Grouping` is what we use elsewhere. Also s/group-by//, this isn't SQL.